ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/BlobSidecarValidator.java

```diff
@@ -201,13 +201,48 @@ private boolean isSignatureValidWithRespectToProposerIndex(
 
     return signatureValid
         && receivedValidBlobSidecarInfoSet.add(
-            new SlotAndBlockRoot(
-                signedBlobSidecar.getBlobSidecar().getSlot(),
+            new IndexAndBlockRoot(
+                signedBlobSidecar.getBlobSidecar().getIndex(),
                 signedBlobSidecar.getBlobSidecar().getBlockRoot()));
   }
 
-  private boolean blobSidecarIsFirstWithValidSignatureForSlot(BlobSidecar blobSidecar) {
+  private boolean isFirstWithValidSignatureForIndexAndBlockRoot(BlobSidecar blobSidecar) {
     return !receivedValidBlobSidecarInfoSet.contains(
-        new SlotAndBlockRoot(blobSidecar.getSlot(), blobSidecar.getBlockRoot()));
+        new IndexAndBlockRoot(blobSidecar.getIndex(), blobSidecar.getBlockRoot()));
+  }
+
+  static class IndexAndBlockRoot {
```

Comment:
I think would be useful to move this object to spec.blobs similar to `SlotAndBlockRoot`, so can reuse it for the future.

Reply:
Actually we can also just use `BlobIdentifier`?

Reply:
If i have to move it next to SlotAndBlockRoot I'd be identical to it..I'd be tempted to name it UInt64AndBytes32 and having the meaningful information on the var name...

Reply:
let's leave it as it is in this PR, can move it if we need it somewhere else at some point